seqr/views/utils/file_utils_tests.py

```diff
@@ -9,6 +10,9 @@
 from seqr.views.utils.file_utils import save_temp_file, parse_file, load_uploaded_file
 from seqr.views.utils.test_utils import AuthenticationTestCase
 
+import openpyxl as xl
+from tempfile import NamedTemporaryFile
```

Comment:
can you move these imports up so the app specific imports are all below the external imports

Reply:
ok